core/src/apps/webauthn/fido2.py

```diff
@@ -1036,7 +1036,7 @@ async def keepalive_loop(self) -> None:
         try:
             if not state:
                 return
-            while utime.ticks_ms() < self.deadline:
+            while utime.ticks_diff(utime.ticks_ms(), self.deadline) > 0:
```

Comment:
Shouldn't this be

```suggestion
            while utime.ticks_diff(utime.ticks_ms(), self.deadline) < 0:
```
?


because:

`A < B` -> `A - B < 0`

Reply:
Yeah, I agree with @prusnak, see https://docs.micropython.org/en/v1.15/library/utime.html#utime.ticks_diff.
It can also be written as:
```py
while utime.ticks_diff(self.deadline, utime.ticks_ms()) > 0
```
but I have no personal preference either way.

Reply:
I also agree with the suggestions above